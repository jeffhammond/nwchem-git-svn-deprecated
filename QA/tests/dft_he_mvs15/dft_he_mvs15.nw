start dft_he_mvs15

geometry 
 he 0 0 0
 symmetry oh
end

basis
 * library cc-pvtz
end

dft
 xc xmvs15
end

task dft