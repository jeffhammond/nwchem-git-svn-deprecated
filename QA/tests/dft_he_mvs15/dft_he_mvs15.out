 argument  1 = dft_he_mvs15.nw
                                         
                                         
 
 
              Northwest Computational Chemistry Package (NWChem) 6.5
              ------------------------------------------------------
 
 
                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352
 
                              Copyright (c) 1994-2013
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute
 
             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file
 
                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = lagrange
    program         = /home/edo/nwchem/bin/LINUX64/nwchem
    date            = Mon Oct  5 18:15:01 2015

    compiled        = Fri_Sep_25_12:58:00_2015
    source          = /home/edo/nwchem
    nwchem branch   = Development
    nwchem revision = 27477
    ga revision     = 10586
    input           = dft_he_mvs15.nw
    prefix          = dft_he_mvs15.
    data base       = ./dft_he_mvs15.db
    status          = startup
    nproc           =        1
    time left       =     -1s



           Memory information
           ------------------

    heap     =   13107198 doubles =    100.0 Mbytes
    stack    =   13107195 doubles =    100.0 Mbytes
    global   =   26214400 doubles =    200.0 Mbytes (distinct from heap & stack)
    total    =   52428793 doubles =    400.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------
 
  0 permanent = .
  0 scratch   = .
 
 
 
 
                                NWChem Input Module
                                -------------------
 
 

 Scaling coordinates for geometry "geometry" by  1.889725989
 (inverse scale =  0.529177249)

 
 
                             Geometry "geometry" -> ""
                             -------------------------
 
 Output coordinates in angstroms (scale by  1.889725989 to convert to a.u.)
 
  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 he                   2.0000     0.00000000     0.00000000     0.00000000
 
      Atomic Mass 
      ----------- 
 
      he                 4.002600
 

 Effective nuclear repulsion energy (a.u.)       0.0000000000

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000
 
      Symmetry information
      --------------------
 
 Group name              Oh       
 Group number             44
 Group order              48
 No. of unique centers     1
 
      Symmetry unique atoms
 
     1
 
 
            XYZ format geometry
            -------------------
     1
 geometry
 he                    0.00000000     0.00000000     0.00000000
 


 Summary of "ao basis" -> "" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 *                          cc-pvtz                   on all atoms 


 
                                 NWChem DFT Module
                                 -----------------
 
 
                      Basis "ao basis" -> "ao basis" (cartesian)
                      -----
  he (Helium)
  -----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  2.34000000E+02  0.002587
  1 S  3.51600000E+01  0.019533
  1 S  7.98900000E+00  0.090998
  1 S  2.21200000E+00  0.272050
 
  2 S  6.66900000E-01  1.000000
 
  3 S  2.08900000E-01  1.000000
 
  4 P  3.04400000E+00  1.000000
 
  5 P  7.58000000E-01  1.000000
 
  6 D  1.96500000E+00  1.000000
 


 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 he                         cc-pvtz                  6       15   3s2p1d




 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 he                         cc-pvtz                  6       15   3s2p1d


      Symmetry analysis of basis
      --------------------------
 
        a1g         4
        a1u         0
        a2g         0
        a2u         0
        eg          2
        eu          0
        t1g         0
        t1u         6
        t2g         3
        t2u         0
 
  Caching 1-el integrals 
 
            General Information
            -------------------
          SCF calculation type: DFT
          Wavefunction type:  closed shell.
          No. of atoms     :     1
          No. of electrons :     2
           Alpha electrons :     1
            Beta electrons :     1
          Charge           :     0
          Spin multiplicity:     1
          Use of symmetry is: on ; symmetry adaption is: on 
          Maximum number of iterations:  30
          AO basis - number of functions:    15
                     number of shells:     6
          Convergence on energy requested: 1.00D-06
          Convergence on density requested: 1.00D-05
          Convergence on gradient requested: 5.00D-04
 
              XC Information
              --------------
                   MVS metaGGA Exchange Functional  1.000          
 
             Grid Information
             ----------------
          Grid used for XC integration:  medium    
          Radial quadrature: Mura-Knowles        
          Angular quadrature: Lebedev. 
          Tag              B.-S. Rad. Rad. Pts. Rad. Cut. Ang. Pts.
          ---              ---------- --------- --------- ---------
          he                  0.35       49           4.0       434
          Grid pruning is: on 
          Number of quadrature shells:    49
          Spatial weights used:  Erf1
 
          Convergence Information
          -----------------------
          Convergence aids based upon iterative change in 
          total energy or number of iterations. 
          Levelshifting, if invoked, occurs when the 
          HOMO/LUMO gap drops below (HL_TOL): 1.00D-02
          DIIS, if invoked, will attempt to extrapolate 
          using up to (NFOCK): 10 stored Fock matrices.

                    Damping( 0%)  Levelshifting(0.5)       DIIS
                  --------------- ------------------- ---------------
          dE  on:    start            ASAP                start   
          dE off:    2 iters         30 iters            30 iters 

 
      Screening Tolerance Information
      -------------------------------
          Density screening/tol_rho: 1.00D-10
          AO Gaussian exp screening on grid/accAOfunc:  14
          CD Gaussian exp screening on grid/accCDfunc:  20
          XC Gaussian exp screening on grid/accXCfunc:  20
          Schwarz screening/accCoul: 1.00D-08

 
      Superposition of Atomic Density Guess
      -------------------------------------
 
 Sum of atomic energies:          -2.86115334
 
      Non-variational initial energy
      ------------------------------

 Total energy =      -2.861153
 1-e energy   =      -3.887056
 2-e energy   =       1.025903
 HOMO         =      -0.917625
 LUMO         =       0.590162
 
 
      Symmetry analysis of molecular orbitals - initial
      -------------------------------------------------
 

 !! scf_movecs_sym_adapt:    5 vectors were symmetry contaminated

  Symmetry fudging

 !! scf_movecs_sym_adapt:    5 vectors were symmetry contaminated

  Numbering of irreducible representations: 
 
     1 a1g         2 a1u         3 a2g         4 a2u         5 eg      
     6 eu          7 t1g         8 t1u         9 t2g        10 t2u     
 
  Orbital symmetries:
 
     1 a1g         2 a1g         3 t1u         4 t1u         5 t1u     
     6 a1g         7 t2g         8 t2g         9 t2g        10 eg      
    11 eg      
 
   Time after variat. SCF:      0.0
   Time prior to 1st pass:      0.0

 #quartets = 2.310D+02 #integrals = 1.797D+03 #direct =  0.0% #cached =100.0%


 Integral file          = ./dft_he_mvs15.aoints.0
 Record size in doubles =  65536        No. of integs per rec  =  43688
 Max. records in memory =      2        Max. records in file   = 403472
 No. of bits per label  =      8        No. of bits per value  =     64


 Grid_pts file          = ./dft_he_mvs15.gridpts.0
 Record size in doubles =  12289        No. of grid_pts per rec  =   3070
 Max. records in memory =      9        Max. recs in file   =   2151679


 !! scf_movecs_sym_adapt:    5 vectors were symmetry contaminated

  Symmetry fudging

 !! scf_movecs_sym_adapt:    5 vectors were symmetry contaminated


           Memory utilization after 1st SCF pass: 
           Heap Space remaining (MW):       12.86            12864494
          Stack Space remaining (MW):       13.11            13106988

   convergence    iter        energy       DeltaE   RMS-Dens  Diis-err    time
 ---------------- ----- ----------------- --------- --------- ---------  ------
 d= 0,ls=0.0,diis     1     -2.8663596397 -2.87D+00  4.14D-03  1.06D-02     0.1

 !! scf_movecs_sym_adapt:    4 vectors were symmetry contaminated

  Symmetry fudging

 !! scf_movecs_sym_adapt:    2 vectors were symmetry contaminated

 d= 0,ls=0.0,diis     2     -2.8670652072 -7.06D-04  8.69D-04  1.62D-04     0.1

 !! scf_movecs_sym_adapt:    4 vectors were symmetry contaminated

  Symmetry fudging

 !! scf_movecs_sym_adapt:    3 vectors were symmetry contaminated

 d= 0,ls=0.0,diis     3     -2.8670789994 -1.38D-05  7.67D-05  8.68D-07     0.1

 !! scf_movecs_sym_adapt:    5 vectors were symmetry contaminated

  Symmetry fudging

 !! scf_movecs_sym_adapt:    5 vectors were symmetry contaminated

 d= 0,ls=0.0,diis     4     -2.8670790606 -6.12D-08  3.93D-06  1.56D-08     0.1


         Total DFT energy =       -2.867079060595
      One electron energy =       -3.880098709943
           Coulomb energy =        2.039409906766
    Exchange-Corr. energy =       -1.026390257419
 Nuclear repulsion energy =        0.000000000000

 Numeric. integr. density =        1.999999973276

     Total iterative time =      0.1s


 
                  Occupations of the irreducible representations
                  ----------------------------------------------
 
                     irrep           alpha         beta
                     --------     --------     --------
                     a1g               1.0          1.0
                     a1u               0.0          0.0
                     a2g               0.0          0.0
                     a2u               0.0          0.0
                     eg                0.0          0.0
                     eu                0.0          0.0
                     t1g               0.0          0.0
                     t1u               0.0          0.0
                     t2g               0.0          0.0
                     t2u               0.0          0.0
 
 
                       DFT Final Molecular Orbital Analysis
                       ------------------------------------
 
 Vector    1  Occ=2.000000D+00  E=-5.880584D-01  Symmetry=a1g
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 3.4D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      0.446970  1 He s                  1      0.364054  1 He s          
     3      0.332438  1 He s          
 
 Vector    2  Occ=0.000000D+00  E= 6.064213D-01  Symmetry=a1g
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 1.5D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      1.601256  1 He s                  2     -1.454294  1 He s          
 
 Vector    3  Occ=0.000000D+00  E= 1.347888D+00  Symmetry=t1u
              MO Center= -3.2D-35, -1.8D-34,  2.5D-35, r^2= 4.7D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8      0.779358  1 He py                 7      0.643951  1 He px         
 
 Vector    4  Occ=0.000000D+00  E= 1.347888D+00  Symmetry=t1u
              MO Center=  3.5D-17,  7.0D-17,  4.8D-17, r^2= 4.7D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     9      0.896184  1 He pz                 7     -0.441547  1 He px         
     8      0.206928  1 He py         
 
 Vector    5  Occ=0.000000D+00  E= 1.347888D+00  Symmetry=t1u
              MO Center=  2.7D-19, -1.8D-19, -1.9D-19, r^2= 4.7D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     7      0.656728  1 He px                 8     -0.625069  1 He py         
     9      0.467896  1 He pz         
 
 Vector    6  Occ=0.000000D+00  E= 3.478626D+00  Symmetry=a1g
              MO Center=  2.0D-18, -9.2D-19, -4.3D-18, r^2= 8.8D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      4.000398  1 He s                  1     -1.711171  1 He s          
     3     -1.362549  1 He s                 10     -0.717619  1 He dxx        
    13     -0.717619  1 He dyy               15     -0.717619  1 He dzz        
 
 Vector    7  Occ=0.000000D+00  E= 6.314609D+00  Symmetry=t2g
              MO Center= -2.8D-17, -5.5D-17,  1.7D-17, r^2= 2.5D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    11      1.433293  1 He dxy               12     -0.888792  1 He dxz        
    14     -0.394614  1 He dyz        
 
 Vector    8  Occ=0.000000D+00  E= 6.314609D+00  Symmetry=t2g
              MO Center= -9.1D-18, -1.0D-18,  2.9D-18, r^2= 2.5D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    12      1.275998  1 He dxz               11      0.971411  1 He dxy        
    14      0.654363  1 He dyz        
 
 Vector    9  Occ=0.000000D+00  E= 6.314609D+00  Symmetry=t2g
              MO Center=  1.0D-17, -2.1D-17, -9.6D-18, r^2= 2.5D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    14      1.554377  1 He dyz               12     -0.762810  1 He dxz        
 
 Vector   10  Occ=0.000000D+00  E= 6.314609D+00  Symmetry=eg
              MO Center= -1.6D-17,  4.9D-18, -5.3D-17, r^2= 2.5D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    15      0.992973  1 He dzz               10     -0.598975  1 He dxx        
    13     -0.393997  1 He dyy        
 
 Vector   11  Occ=0.000000D+00  E= 6.314609D+00  Symmetry=eg
              MO Center=  5.6D-18,  3.0D-18, -1.7D-18, r^2= 2.5D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    13      0.919112  1 He dyy               10     -0.800768  1 He dxx        
 

 Parallel integral file used       1 records with       0 large values


 Task  times  cpu:        0.1s     wall:        0.2s
 
 
                                NWChem Input Module
                                -------------------
 
 
 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:  170      170     5138     1830      912        0        0      137     
number of processes/call 1.00e+00 1.00e+00 1.00e+00 0.00e+00 0.00e+00
bytes total:             1.37e+06 4.19e+05 1.64e+06 0.00e+00 0.00e+00 1.10e+03
bytes remote:            0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 70200 bytes
 
MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        24	        55
	current total bytes		         0	         0
	maximum total bytes		   1941640	  22510264
	maximum total K-bytes		      1942	     22511
	maximum total M-bytes		         2	        23
 
 
                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:
 
                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018
 
                                      AUTHORS
                                      -------
          E. Apra, E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski,
       T. P. Straatsma, M. Valiev, H. J. J. van Dam, D. Wang, T. L. Windus,
        J. Hammond, J. Autschbach, K. Bhaskaran-Nair, J. Brabec, K. Lopata,
        S. Krishnamoorthy, W. Ma, M. Klemm, O. Villa, Y. Chen, V. Anisimov,
    F. Aquino, S. Hirata, M. T. Hackler, T. Risthaus, M. Malagoli, A. Marenich,
   A. Otero-de-la-Roza, J. Mullin, P. Nichols, R. Peverati, J. Pittner, Y. Zhao,
          P.-D. Fan, A. Fonari, M. Williamson, R. J. Harrison, M. Dupuis,
     D. Silverstein, D. M. A. Smith, J. Nieplocha, V. Tipparaju, M. Krishnan,
       B. E. Van Kuiken, A. Vazquez-Mayagoitia, L. Jensen, M. Swart, Q. Wu,
   T. Van Voorhis, A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown, G. Cisneros,
     G. I. Fann, H. Fruchtl, J. Garza, K. Hirao, R. A. Kendall, J. A. Nichols,
       K. Tsemekhman, K. Wolinski, J. Anchell, D. E. Bernholdt, P. Borowski,
       T. Clark, D. Clerc, H. Dachsel, M. J. O. Deegan, K. Dyall, D. Elwood,
      E. Glendening, M. Gutowski, A. C. Hess, J. Jaffe, B. G. Johnson, J. Ju,
        R. Kobayashi, R. Kutteh, Z. Lin, R. Littlefield, X. Long, B. Meng,
      T. Nakajima, S. Niu, L. Pollack, M. Rosing, K. Glaesemann, G. Sandrone,
      M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe, A. T. Wong, Z. Zhang.

 Total times  cpu:        0.1s     wall:        0.3s
